src/web/app/pages-admin/admin-search-page/admin-search-page.component.spec.ts

```diff
@@ -36,10 +40,11 @@ describe('AdminSearchPageComponent', () => {
         FormsModule,
         HttpClientTestingModule,
         NgbTooltipModule,
+        BrowserAnimationsModule,
       ],
       providers: [AccountService, SearchService, StatusMessageService, NgbModal],
     })
-    .compileComponents();
+        .compileComponents();
```

Comment:
Any particular reason for this extra indent?

Reply:
I think those were auto-indented by my IDE. It did managed to pass the lint test so I thought it was fine. Nevertheless, I will double check my IDE settings again for indentation. Thanks for pointing out!